presto-orc/src/test/java/com/facebook/presto/orc/TestOrcReaderPositions.java

```diff
@@ -160,6 +174,25 @@ public void testRowGroupSkipping()
         }
     }
 
+    @Test
+    public void testReadUserMetadata()
+            throws Exception
+    {
+        try (TempFile tempFile = new TempFile()) {
+            Map<String, String> metadata = ImmutableMap.of(
+                    "a", "ala",
+                    "b", "ma",
+                    "c", "kota");
+            createFileWithJustUserMetadata(tempFile.getFile(), metadata);
+
+            OrcDataSource orcDataSource = new FileOrcDataSource(tempFile.getFile(), new DataSize(1, DataSize.Unit.MEGABYTE), new DataSize(1, DataSize.Unit.MEGABYTE), new DataSize(1, DataSize.Unit.MEGABYTE));
```

Comment:
I think I considered to use that. But this is in presto-raptor. And TestOrcReaderPositions is in presto-orc. So I do not have access to that.

It is not even easy to change as it would have to be in presto-orc/main source path.
To be available by both tests in presto-orc and presto-raptor. But possibly it cannot be in main as it will be lacking test testNG libraries and other test stuff.

Any ideas?